src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -151,6 +151,28 @@ public static void AddReadOnlyThrows()
         }
 
         [Fact]
+        public static void AddDisposedThrowsArgumentNullException()
+        {
+            using (X509Store store = new X509Store(StoreName.My, StoreLocation.CurrentUser))
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                store.Open(OpenFlags.ReadWrite);
+
+                using (var coll = new ImportedCollection(store.Certificates))
+                {
+                    // Add only throws when it has to do work.  If, for some reason, this certificate
+                    // is already present in the CurrentUser\My store, we can't really test this
+                    // functionality.
+                    if (!coll.Collection.Contains(cert))
```

Comment:
Since the if we're testing is in preconditions, I don't think that reading the store certs collection and doing the contains test is relevant.